### PR TITLE
Upped Infinity Drill Mining Level

### LINKED ISSUE #1079: Infinity Drill has too low mining level
**Modpack version**
1.59a

**Can the issue be reproduced?**
Yes

**Issue**
Infinity Drill appears to have a mining level of 3, not enough to actually harvest emerald ore and others.

### PR DESCRIPTION
**Pull request by MaxNeedsSnacks**
(This closes #1079, it's currently equivalent to Osgloglas, let me know if this needs changing)

### CODE REVIEW

**Comment by NillerMedDild** [comment]
Looks good to me :D

**Comment by MaxNeedsSnacks** [comment]
Oh yeah by the way, if you don't want the extra "merged commit" you can rebase and merge so it only rebases the branch to the latest commit and doesn't add the annoying "Merged branch into branch" commit

**Comment by NillerMedDild** [comment]
Oh cool.

**Comment by Fargoth8624** [comment]
I just checked this in 1.62 and It was still showing mining level 3.

**Comment by MaxNeedsSnacks** [comment]
https://github.com/NillerMedDild/Enigmatica2Expert/issues/1079#issuecomment-482968886

TL;DR It'll be in 1.63, we forgot to merge this lol
